fdbserver/storageserver.actor.cpp

```diff
@@ -12163,10 +12163,26 @@ ACTOR Future<Void> updateStorage(StorageServer* data) {
 
 		recentCommitStats.back().whenCommit = now();
 		try {
-			wait(ioTimeoutErrorIfCleared(durable,
-			                             SERVER_KNOBS->MAX_STORAGE_COMMIT_TIME,
-			                             data->getEncryptCipherKeysMonitor->degraded(),
-			                             "StorageCommit"));
+			loop {
+				choose {
+					when(wait(ioTimeoutErrorIfCleared(durable,
```

Comment:
It seems `ioTimeoutErrorIfCleared` is already emitting trace events, like `IoDegraded` and `IoTimeoutError`. Is the idea here for gaterhing fetch key progress?